.azure-pipelines-templates/model_checking.yml

```diff
@@ -24,14 +24,25 @@ jobs:
         workingDirectory: tla
         displayName: MCccfraft.cfg
 
-      - script: ./tlc.sh -workers auto -config consensus/MCccfraftAtomicReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftAtomicReconfig.trace.tla -dumpTrace json MCccfraftAtomicReconfig.json
+      - script: ./tlc.sh -workers auto -config consensus/MCccfraftAtomicReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftAtomicReconfig.trace.tla -dumpTrace json MCccfraftAtomicReconfig.json -coverage 1 -nowarning
         workingDirectory: tla
         displayName: MCccfraftAtomicReconfig.cfg
 
-      - script: ./tlc.sh -workers auto -config consensus/MCccfraftWithReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftWithReconfig.trace.tla -dumpTrace json MCccfraftWithReconfig.json
+      - script: ./tlc.sh -workers auto -config consensus/MCccfraftWithReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftWithReconfig.trace.tla -dumpTrace json MCccfraftWithReconfig.json -coverage 1 -nowarning
         workingDirectory: tla
         displayName: MCccfraftWithReconfig.cfg
 
+      - script: |
```

Comment:
It looks like coverage runs are about 2x slower than normal, so while this is interesting as a one-off experiment, I do not want to merge it. The model checking is slow enough as it is.

Reply:
That's about the expected slowdown.  Perhaps, we can collect coverage data every nth run?

Reply:
Ideally we'd have a daily run, and a way to distinguish it enough in the yaml that we can parametrise the tasks to only do coverage then. Let's see how much of a pain that is.